.github/workflows/integration_test_charm.yaml

```
@@ -360,6 +361,20 @@ jobs:
           mkdir ~/.kube/
           # Used by lightkube and kubernetes (Python package)
           sg '${{ steps.parse-cloud.outputs.group }}' -c "microk8s config > ~/.kube/config"
+      # Temporarily collect logs to debug intermittent issue
+      # https://github.com/canonical/data-platform-workflows/issues/217
+      - name: Collect microk8s logs for debugging
+        timeout-minutes: 5
+        if: ${{ failure() && steps.microk8s-setup.outcome == 'failure' }}
+        run: sudo microk8s inspect
+      - name: Upload microk8s logs for debugging
+        timeout-minutes: 5
+        if: ${{ failure() && steps.microk8s-setup.outcome == 'failure' }}
+        uses: actions/upload-artifact@v4
+        with:
+          name: debug-logs-microk8s-intergration-test-charm-${{ inputs.cloud }}-juju-${{ inputs.juju-agent-version || steps.parse-versions.outputs.snap_channel_for_artifact }}-${{ inputs.architecture }}-${{ matrix.groups.artifact_group_id }}
```

This will need to be changed in the step name, too, in another PR.
```suggestion
          name: debug-logs-microk8s-integration-test-charm-${{ inputs.cloud }}-juju-${{ inputs.juju-agent-version || steps.parse-versions.outputs.snap_channel_for_artifact }}-${{ inputs.architecture }}-${{ matrix.groups.artifact_group_id }}
```

---

It was a mistake. I thought I saw this in a step name. Now I realise that I saw it only in the places you fixed it.